soknad/src/main/java/no/nav/k9/søknad/ytelse/pls/v1/Pleietrengende.java

```diff
@@ -50,15 +54,21 @@ public PersonIdent getPersonIdent() {
     }
 
     public Pleietrengende medNorskIdentitetsnummer(NorskIdentitetsnummer norskIdentitetsnummer) {
-        this.norskIdentitetsnummer = norskIdentitetsnummer;
+        this.norskIdentitetsnummer = Objects.requireNonNull(norskIdentitetsnummer, "norskIdentitetsnummer");
         return this;
     }
 
     public Pleietrengende medFødselsdato(LocalDate fødselsdato) {
-        this.fødselsdato = fødselsdato;
+        this.fødselsdato = Objects.requireNonNull(fødselsdato, "fødselsdato");
         return this;
     }
 
+    @AssertTrue(message = "norskIdentitetsnummer eller fødselsdato må være satt")
+    private boolean isOk() {
+        return (norskIdentitetsnummer != null && norskIdentitetsnummer.getVerdi() != null)
```

Comment:
Hva gjør du hvis begge to er satt? Det bør bare være lov å sette en av dem om gangen.. Alternativt så må vi da alltid prioritere ident foran dato som forsåvidt også kan være akseptabelt men kan være uheldig hvis det er opplysninger om et annen person også

Reply:
Hvorfor skulle det være opplysninger fra flere personer i pleietrengende? Er det en reell scenario?

Reply:
Søker skal uansett oppgi enten norskIdentitetsnummer eller fødselsdato i dialogen, så jeg er egentlig enig i at det ikke burde være mulig å sette begge :)